app/controllers/StripeController.scala

```diff
@@ -129,7 +133,11 @@ class StripeController(paymentServices: PaymentServices, stripeConfig: Config, o
         .addingToSession("amount" -> contributionAmount.show)
         .setCookie[ContribTimestampCookieAttributes](Instant.ofEpochSecond(charge.created).toString)
     }.recover {
-      case e: Stripe.Error => BadRequest(Json.toJson(e))
+      case e: Stripe.Error => {
```

Comment:
I'm wondering if we want to recover from any non-fatal error here? In the context of this pull request it'd mean we'd log any Stripe payment failure with Cloud Watch. More generally, we'd send a specific error to the client instead of an Internal Server Error (which I don't think is currently handled).